reference/json/BigCommerce_Server_to_Server_Cart_API.oas2.json

```diff
@@ -3693,7 +3694,7 @@
           "readOnly": true
         },
         "coupon_type": {
```

Comment:
If this can only be one of 5 strings, we should make it an enum and specify the acceptable values in the array instead of it being a string and putting the accepted values in the description.